### ISSUE TITLE
address input ignored when setting LB IP

### ISSUE THREAD

**Issue by jkurban** [terraform-google-modules/terraform-google-lb-http#230]
### TL;DR

I am setting the "address" input to a static global external IPV4 address that I created in previous terraform step. However when it runs, it generates a brand new IP address and assigns that to the forwarding rules instead of the static global IP created.

...
  ssl                             = true
  managed_ssl_certificate_domains = [var.mydomain.com]
  https_redirect                  = true
  address                         = google_compute_global_address.default.address
...

### Expected behavior

I would expect the static global IP address, created in the previous terraform step and applied using the address input variable, to be assigned to the load balancer. Instead it just creates another new IP address and assigns that to the load balancer.

### Observed behavior

it just creates another new IP address and assigns that to the load balancer instead of using the global static IP.

### Terraform Configuration

```hcl
...
resource "google_compute_global_address" "default" {
  name          = "lb-global-static-ip"
  ip_version    = "IPV4"
}

module "lb-http" {
  source            = "GoogleCloudPlatform/lb-http/google//modules/serverless_negs"
  version           = "~> 4.4"

  project           = var.project_id
  name              = var.lb_name

  ssl                             = true
  managed_ssl_certificate_domains = [var.subdomain]
  https_redirect                  = true
  address                         = google_compute_global_address.default.address
  backends = {
    default = {
      description                     = "serverless function endpoint"
      enable_cdn                      = false
      custom_request_headers          = null
      custom_response_headers         = null
      security_policy                 = google_compute_security_policy.policy.id


      log_config = {
        enable = true
        sample_rate = 1.0
      }

      groups = [
        {
          # Your serverless service should have a NEG created that's referenced here.
          group = google_compute_region_network_endpoint_group.function_neg.id
        }
      ]

      iap_config = {
        enable               = false
        oauth2_client_id     = null
        oauth2_client_secret = null
      }
    }
  }
  depends_on = [
    google_compute_global_address.default,
  ]  
}
...
```

### Terraform Version

```sh
v1.1.4
```

### Additional information

I love this module! Saves so much time. I just may be thinking the address input is something that it isn't.

**Comment by morgante** [comment]
You need to set `create_address = false` to reuse an existing address.

**Comment by jkurban** [comment]
Well, I feel stupid. Thats the only thing I didn't try. And the doc even says default it true. Thank you. Its working nicely now. Again, this is an awesome module!

**Comment by morgante** [comment]
Glad it's proving helpful!

**Comment by AndreMiras** [comment]
Same thing happened to me with`create_url_map` recently.
Maybe the module could automatically disable it using a ternary when `address` was provided and same thing with `url_map`. Or at least issue a warning (if possible) when both are provided
